### PR TITLE
docs: Update issue templates

### PR DESCRIPTION
**Pull request by xopham**
Standardization of issue creation.

Good point to have a discussion, whether we want issue templates. Please share your thoughts.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
```
Comment (peterthomassen) [inline .github/ISSUE_TEMPLATE/bug_report.md:18]: Pre-filled lines increase work for the reporter. I'd prefer

```
1. 
2. 
3. 
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
```
Comment (peterthomassen) [inline .github/ISSUE_TEMPLATE/bug_report.md:24]: uncommon

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
```
Comment (peterthomassen) [inline .github/ISSUE_TEMPLATE/bug_report.md:29]: How does the browser matter? Shouldn't we be asking for the kubectl version, the cluster type (Minikube, ...), and the Connaisseur version / commit sha?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+**Smartphone (please complete the following information):**
+ - Device: [e.g. iPhone6]
+ - OS: [e.g. iOS8.1]
+ - Browser [e.g. stock browser, safari]
+ - Version [e.g. 22]
```
Comment (peterthomassen) [inline .github/ISSUE_TEMPLATE/bug_report.md:35]: How is that relevant?

**Review by peterthomassen — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
```
Comment (Starkteetje) [inline .github/ISSUE_TEMPLATE/feature_request.md:10]: Would ask first for a description of the feature itself. Only then am I interested in a related problem to see the use-case. Feature could be relevant without the use-case being solved via the feature

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
```
Comment (Starkteetje) [inline .github/ISSUE_TEMPLATE/feature_request.md:13]: Would not make solution/alternatives so prominent. Depending on who raises the ticket, they may have ideas or not on how to solve. Would rather go for "Optional: Solution ideas (if you have any)" at the very end

**Review by Starkteetje — COMMENTED** [review]
For bug report, I can see a template being useful to ensure we get enough info to pinpoint a problem. For feature I'm not convinced of either necessity nor usefulness, tbh

**Review by peterthomassen — APPROVED** [review]

**Review by peterthomassen — DISMISSED** [review]
lgtm

**Review by Starkteetje — APPROVED** [review]
